git_machete/exceptions.py

```diff
@@ -26,3 +28,11 @@ def __init__(self, msg: str) -> None:
 
     def __str__(self) -> str:
         return str(self.msg)
+
+
+class ExitCode(IntEnum):
+    SUCCESS = 0
+    ARGUMENT_ERROR = 1
+    MACHETE_EXCEPTION = 2
```

Comment:
I'd rather swap these: most CLIs use 1 for incorrect result (e.g. `grep` when no match found), and 2 for invalid command-line arguments